packages/webapp/pages/squads/index.tsx

```diff
@@ -43,8 +54,9 @@ const SquadsPage = (): ReactElement => {
       }),
     {
       getNextPageParam: (lastPage) =>
-        lastPage?.notifications?.pageInfo?.hasNextPage &&
-        lastPage?.notifications?.pageInfo?.endCursor,
```

Comment:
I think this was actually a typo so updated

Reply:
Yep, well spotted 👀